oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/dsl/SampleFamily.java

```diff
@@ -233,12 +234,29 @@ public SampleFamily tag(Closure<?> cl) {
         }).toArray(Sample[]::new));
     }
 
+    /**
+     * Ignore histogram decrease bucket value, at agent side meter, need to ignore decrease histogram bucket value
+     */
+    public SampleFamily ignoreHistogramDecrease() {
```

Comment:
From https://en.wikipedia.org/wiki/Histogram#Mathematical_definition, there are two types of histogram bin:
 * Cumulative: prometheus, otel.
 * Ordinary: agent meter.

Let's provide two functions to set it, and defaults to cumulative.

Reply:
What I mean is `histogramDecrease` is not clear and very confusing to ppl. Let's opt for the notions that is well-known for ppl.

Reply:
Created `HistogramType` enum to adapt this.